pkg/backup/file_backend_test.go

```diff
@@ -22,6 +22,49 @@ import (
 	"testing"
 )
 
+func TestFileBackendGetLatest(t *testing.T) {
+	names := []string{
+		makeBackupName("3.0.1", 12),
```

Comment:
we cannot. we need to ensure:

1. we ignore bad name
2. we ignore bad suffix
3. we ignore version
4. we get the file with large rev